common/test/model/MetaDataTest.scala

```diff
@@ -159,10 +150,10 @@ class MetaDataTest extends FlatSpec with Matchers {
 
   it should "show https Facebook og:url for content first published after our decision to start advertisng https canonical urls to Facebook" in {
     val content = contentApi(
-      publicationDate = dateAfterWeStartedAdvertistingHttpsUrlsToFacebook,
-      firstPublicationDate = Some(dateAfterWeStartedAdvertistingHttpsUrlsToFacebook),
+      publicationDate = Chronos.javaLocalDateTimeToJodaDateTime(dateAfterWeStartedAdvertistingHttpsUrlsToFacebook),
+      firstPublicationDate =
+        Some(Chronos.javaLocalDateTimeToJodaDateTime(dateAfterWeStartedAdvertistingHttpsUrlsToFacebook)),
```

Comment:
Not brave enough to try and migrate the meta date fields in this PR.
My new firstPublished field is Joda like the existing date field; thought some class level consistency was useful.